server/server.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import useWebpackMiddleware from '../helpers/useWebpackMiddleware';
```

Comment:
Ah just another small change. This is technically not a helper so it should be a folder with the server called `environment` or `middleware`. Though in this case,  your function isn't a middleware so prob environment is a better choice for this case.

Reply:
Okay, that makes sense. Moved `useWebpackMiddleware` to server environment folder.